src/demo/components/ExampleCard/ExampleCard.scss

```diff
@@ -22,7 +22,7 @@
   bottom: 0;
   right: 0;
 
-  .codeButton {
+  .codeButton.codeButton {
```

Comment:
huh... this is really weird.

1. shouldn't this be something like .ExampleCard-codeButton ? 
2. If it needs to be more specific should it be .ms-Button.ExampleCard-codeButton ?